platform/darwin/src/MGLStyleValue_Private.h

```diff
@@ -45,6 +48,18 @@ NS_INLINE mbgl::style::TransitionOptions MGLOptionsFromTransition(MGLTransition
     return options;
 }
 
+NS_INLINE std::unique_ptr<mbgl::style::expression::Expression> MGLClusterPropertyFromNSExpression(NSExpression *expression) {
```

Comment:
What's the reason for marking this inline? Could this live in the .mm file?

Reply:
cc @zmiao

Reply:
It can be removed, and move the definition to .mm file